blotoutSDK/src/main/java/com/analytics/blotout/BlotoutAnalyticsInternal.kt

```diff
@@ -4,22 +4,21 @@ import android.app.Application
 import android.content.Context
 import android.util.Log
 import com.analytics.blotout.model.*
+import com.analytics.blotout.model.CompletionHandler
 import com.analytics.blotout.network.HostConfiguration
 import com.analytics.blotout.repository.EventRepository
 import com.analytics.blotout.repository.impl.SharedPreferenceSecureVaultImpl
 import com.analytics.blotout.util.Constant
 import com.analytics.blotout.util.Errors
 import com.analytics.blotout.util.serializeToMap
-import kotlinx.coroutines.CoroutineScope
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.launch
-import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.*
 import java.lang.Exception
 
 open class BlotoutAnalyticsInternal : BlotoutAnalyticsInterface {
 
     companion object {
         private const val TAG = "AnalyticsInternal"
+        private var isSdkinitiliazed = true
```

Comment:
IMO , if another function is called before the callback of init is processed , we may still face a crash since the default is true.
Hence my suggestion  would be to have default value as false & set it to true in case of init success.

Reply:
Done.